jwala-webservices/src/main/java/com/cerner/jwala/ws/rest/v1/service/jvm/JvmServiceRest.java

```diff
@@ -178,4 +178,21 @@ Response previewResourceTemplate(@ApiParam(value = "The name of the JVM", requir
                                      @ApiParam(value = "The name of the JVM's group", required = true) @MatrixParam("groupName") String groupName,
                                      @ApiParam(value = "The content to be generated", required = true) String template);
 
+    /**
+     * Upgraded the JDK version of the mentioned JVM.
+     * @param jvmName the name of the JVM
+     * @param aUser the user
+     * @return {@link Response}
+     */
+    @PUT
+    @Path("/{jvmName}/upgradeJDK")
+    @ApiOperation(value = "upgrade the JDK version for the JVM",
+            response = Jvm.class
+    )
+    @ApiResponses(@ApiResponse(code = 500, message = "Remote upgradation of JDK failed"))
```

Comment:
This is a very minor thing, but please don't use the word "upgradation" :)
You can just say something like "Attempt to upgrade the JDK failed."